scripts/travisci/check_precommit.sh

```diff
@@ -41,7 +42,7 @@ else
       cat "${commit_msg}"
     fi
 
-  done < <(git rev-list origin/master..."${TRAVIS_BRANCH}")
+  done < <(git rev-list ^origin/master "${TRAVIS_BRANCH}")
```

Comment:
You checked that this doesn't include the head of `origin/master`, right?

Reply:
Yep